test/Microsoft.SqlTools.ServiceLayer.IntegrationTests/SchemaCompare/SchemaCompareServiceTests.cs

```diff
@@ -864,8 +864,10 @@ public async void SchemaCompareIncludeExcludeWithDependencies()
                 t2ExcludeOperation.Execute(TaskExecutionMode.Execute);
                 Assert.False(t2ExcludeOperation.Success, "Excluding Table t2 should fail because view v1 depends on it");
                 Assert.True(t2ExcludeOperation.ComparisonResult.Differences.Where(x => x.SourceObject != null && x.SourceObject.Name.Parts[1] == "t2").First().Included, "Difference Table t2 should still be included because the exclude request failed");
+                Assert.True(t2ExcludeOperation.BlockingDependencies.Count == 1, "There should be one dependency");
+                Assert.True(t2ExcludeOperation.BlockingDependencies[0].SourceValue[1] == "v1",  "Dependency should be View v1");
```

Comment:
not added in this PR but can we please add a constraint in the script for include exclude to cover that for any changes in general